browser/ui/webui/settings/pin_shortcut_handler.cc

```diff
@@ -38,20 +38,63 @@ void PinShortcutHandler::HandleCheckShortcutPinState(
     const base::Value::List& args) {
   AllowJavascript();
 
+  CheckShortcutPinState(false);
```

Comment:
nit: for readers, might worth having comment for the arg

```suggestion
  CheckShortcutPinState(/*from_timer*/ false);
```

Reply:
Added.